osu.Game/Graphics/UserInterface/OsuButton.cs

```diff
@@ -95,10 +92,12 @@ private void load(OsuColour colours)
             if (backgroundColour == null)
                 BackgroundColour = colours.BlueDark;
 
-            Enabled.ValueChanged += enabledChanged;
-            Enabled.TriggerChange();
+            Colour = enablementColour;
+            Enabled.BindValueChanged(_ => this.FadeColour(enablementColour, 200, Easing.OutQuint));
```

Comment:
At this point you might as well move this to `LoadComplete` (to match `OsuAnimatedButton`'s. implementation). It was only in `load` to avoid the initial transition, I believe.

Reply:
That was the intention, moved. It was the bind inside the ctor that previously avoided the initial transition, I believe. Was kinda dodgy..